README.md

```diff
@@ -113,6 +113,24 @@ Elixir's 7 basic types:
 * `lists`
 * `tuples`
 
+### Truthiness in Elixir
```

Comment:
Maybe just `Truthiness` rather than `Truthiness in Elixir` to be in line with the other titles